custom/icds_reports/static/js/directives/program-summary/program-summary.directive.js

```diff
@@ -144,7 +144,9 @@ function ProgramSummaryController($scope, $http, $log, $routeParams, $location,
     });
     $scope.$on('mobile_filter_data_changed', function (event, data) {
         vm.filtersOpen = false;
-        if (data.hasLocation) {
+        if (!data.location) {
+            vm.moveToLocation('national', -1);
+        } else {
```

Comment:
this change is unrelated to most of the PR, but fixes a bug that surfaced while I was working on it